mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/config/WorkerConfiguration.java

```diff
@@ -14,10 +14,9 @@
  * limitations under the License.
  */
 
-package io.mantisrx.server.worker.config;
+package io.mantisrx.server.master.client.config;
```

Comment:
can this also be part of runtime-loader?

Reply:
Moved.